k8s/base/deploy-frontend/deploy.yaml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/master-standalone/deployment-apps-v1.json
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+  labels:
+    app.kubernetes.io/component: frontend
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: frontend
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: frontend
+    spec:
+      securityContext:
+        fsGroup: 1000
+      volumes:
+        - name: dist
+          emptyDir:
+            sizeLimit: 256Mi
+        - name: app-config
+          configMap:
+            name: app-config
+      initContainers:
+        - name: build-files
+          image: lco-education-platform
+          command:
+            - "sh"
+            - "-c"
+            - "cp -r /app/dist/* /app/volume"
```

Comment:
We updated this to simply copy the already built prod files from the docker image to a shared volume on the pod so that the static web server container could serve them. It felt wasteful to always build the app if a new pod is spun up.